ask-sdk-core/src/com/amazon/ask/attributes/AttributesManager.java

```diff
@@ -188,6 +191,74 @@ public void savePersistentAttributes() {
         }
     }
 
+    /**
+     * Deletes the persistent attributes from the persistence layer. An exception is thrown if this method is called when a
+     * {@link PersistenceAdapter} is not configured on the SDK.
+     *
+     * @param keys corresponding values of which needs to be deleted.
+     */
+    public boolean deletePersistentAttributesWithKeys(String... keys) {
```

Comment:
These other delete methods (where the user explicitly provides the keys) are intriguing, but break the abstraction that we're maintaining in the attributes manager. The primary use case here is the ability to delete the _current_ persistent attributes (particularly since they are user or device specific anyway, so deleting beyond those boundaries would be odd/an anti-pattern), and item keys should never be directly exposed to the user here. If a user wants more granular control and the ability to delete multiple items, they'd be better served to use their own DDB client and table.